app/src/pages/inside/stepPage/modals/makeDecisionModal/elements/testItemDetails/testItemDetails.jsx

```diff
@@ -0,0 +1,139 @@
+  noLogsMessage,
+  hideLabels,
+  onClickLinkEvent,
+  isSelected,
+  showErrorLogs,
+}) => {
+  const [showDetails, setShowDetails] = useState(showErrorLogs);
+
+  useEffect(() => {
+    setShowDetails(showErrorLogs);
+  }, [showErrorLogs]);
+
+  const toggleDetails = () => {
+    setShowDetails(!showDetails);
+  };
+
+  return (
+    <div className={cx('test-item-details')}>
+      <ItemHeader
+        item={item}
+        mode={mode}
+        selectItem={selectItem}
+        isSelected={isSelected}
+        hideLabels={hideLabels}
+        onClickLinkEvent={onClickLinkEvent}
+        toggleDetails={toggleDetails}
+      />
+      {showDetails && (
+        <div className={cx('test-item-details-content')}>
+          {item.issue && item.issue.comment && (
+            <div className={cx('defect-comment')}>
+              <div>{Parser(commentIcon)}</div>
+              <div className={cx('defect-comment-content')}>
+                <p className={cx('defect-comment-title')}>Defect Comment</p>
+                <p className={cx('defect-comment-text')}>{item.issue.comment}</p>
+              </div>
+            </div>
+          )}
+          {showDetails &&
+            logs.slice(0, ERROR_LOGS_SIZE).map((log) => (
+              <div key={uniqueId()} className={cx('error-log')}>
+                <StackTraceMessageBlock
+                  level={log.level}
+                  designMode="dark"
+                  maxHeight={90}
+                  eventsInfo={eventsInfo}
+                >
+                  <div className={cx('highlighted-log')}>
+                    {highlightedLogId === log.id && (
+                      <p className={cx('highlighted-log-title')}>
+                        {Parser(asteriskIcon)} {highlightedMessage}
+                      </p>
+                    )}
+                    {log.message}
+                  </div>
+                </StackTraceMessageBlock>
+              </div>
+            ))}
+          {showDetails && !logs.length && (
+            <div className={cx('no-logs')}>
+              <p className={cx('no-logs-text')}>{noLogsMessage}</p>
+            </div>
+          )}
+        </div>
+      )}
+    </div>
+  );
+};
+
+TestItemDetails.propTypes = {
+  item: PropTypes.object.isRequired,
+  isSelected: PropTypes.bool,
+  selectItem: PropTypes.func,
+  nameLink: PropTypes.object,
+  preselected: PropTypes.bool,
+  hideLabels: PropTypes.bool,
+  onClickLinkEvent: PropTypes.func,
+  logs: PropTypes.array,
+  eventsInfo: PropTypes.object,
+  highlightedLogId: PropTypes.number,
+  highlightedMessage: PropTypes.string,
+  noLogsMessage: PropTypes.string,
+  mode: PropTypes.string,
+  showErrorLogs: PropTypes.bool,
+};
+
+TestItemDetails.defaultProps = {
+  isSelected: false,
+  selectItem: () => {},
+  nameLink: {},
+  preselected: false,
+  hideLabels: false,
+  onClickLinkEvent: () => {},
+  logs: [],
+  eventsInfo: {},
+  highlightedLogId: null,
+  highlightedMessage: '',
+  noLogsMessage: '',
+  mode: '',
```

Comment:
Please set default `mode` as `DEFAULT_TEST_ITEM_DETAILS` to avoid specifying it in some cases and provide correct behaviour for component without directly `mode` prop.